src/metadata/metadata_handler.cc

```diff
@@ -170,7 +170,6 @@ std::unique_ptr<MetadataHandler> MetadataHandler::createHandler(const std::share
         return std::make_unique<SubtitleHandler>(config);
     case CH_RESOURCE:
         return std::make_unique<ResourceHandler>(config);
-        break;
```

Comment:
This is no good change. It will raise an exception for each resource!

Reply:
Do you have a suggestion?

Reply:
sorry did not see the `return` in the line above. It's fine!